CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# default code owner for repo
+*        @dhis2/team-platform
+
+# code owners for functionality
+/src/    @dhis2/team-platform @dhis2/team-qa
+/docs/   @dhis2/team-platform @dhis2/team-qa
```

Comment:
Does team platform have to be added explicitly as it'd be excluded otherwise?

Reply:
Yeah, that's how I understand the [docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file).

```
# Order is important; the last matching pattern takes the most
# precedence. When someone opens a pull request that only
# modifies JS files, only @js-owner and not the global
# owner(s) will be requested for a review.
*.js    @js-owner
```